common/djangoapps/third_party_auth/pipeline.py

```diff
@@ -634,7 +636,7 @@ def login_analytics(strategy, *args, **kwargs):
 
 
 @partial.partial
-def change_enrollment(strategy, user=None, *args, **kwargs):
+def change_enrollment(strategy, user=None, is_dashboard=False, is_api=False, *args, **kwargs):
```

Comment:
Nitpick: Something in the docstring to describe is_dashboard and is_api would be helpful